app/src/main/java/org/wikipedia/dataclient/Service.java

```diff
@@ -126,6 +126,9 @@
     @GET(MW_API_PREFIX + "action=query&generator=unreviewedimagelabels&guillimit=10&prop=imagelabels|imageinfo&iiprop=timestamp|user|url|mime|extmetadata&iiurlwidth=" + PREFERRED_THUMB_SIZE)
     @NonNull Observable<MwQueryResponse> getImagesWithUnreviewedLabels(@NonNull @Query("uselang") String lang);
 
+    @GET(MW_API_PREFIX + "action=query&meta=siteinfo&list=watchlist&wllimit=500&wlallrev=1&wlprop=ids|title|flags|comment|parsedcomment|timestamp|sizes|user")
```

Comment:
Noticed there has two `getWatchlist()` to get the `watchlist`, should we update the existing one instead of creating a new one?

Reply:
Whoops, correct!